parm/config/gfs/config.atmanl

```diff
@@ -5,12 +5,17 @@
 
 echo "BEGIN: config.atmanl"
 
-export CASE_ANL=${CASE}
+if [[ ${DOHYBVAR} = "YES" ]]; then
+# shellcheck disable=SC2153
```

Comment:
The shellcheck directive should be indented with the rest of the block

Reply:
shellcheck directive indented at b59662e8.